src/libraries/sendtohelix-browser.targets

```diff
@@ -32,7 +32,8 @@
 
   <PropertyGroup>
     <_ShippingPackagesPath>$([MSBuild]::NormalizeDirectory($(ArtifactsDir), 'packages', $(Configuration), 'Shipping'))</_ShippingPackagesPath>
-    <WorkItemPrefix Condition="'$(WorkItemPrefix)' == ''">$(Scenario)-</WorkItemPrefix>
+    <WorkItemPrefix Condition="'$(WorkItemPrefix)' == '' and '$(WasmEnableThreads)' != 'true'">$(Scenario)-ST-</WorkItemPrefix>
```

Comment:
You already set in `src/libraries/sendtohelix-wasm.targets`

Reply:
I think that's only for `BuildWasmApps` and it happens later, not earlier ?

Reply:
It's included from sendtohelixhelp, so later, but it shouldn't be specific for WBT.

Reply:
The file is imported above this property group. Yeah, the place where you set it in there has a condition for `BuildWasmApps`, but I don't see a reason not to refactor it to a single place